rules/notion_rules/notion_workspace_settings_public_homepage_added.yml

```diff
@@ -0,0 +1,70 @@
+AnalysisType: rule
+Description: A Notion page was set to public in your worksace.
+DisplayName: "Notion Workspace public page added"
+Enabled: true
+Filename: notion_workspace_settings_public_homepage_added.py
+Runbook: A Notion page was made public. Check with the author to determine why this page was made public.
+Severity: Info
+DedupPeriodMinutes: 60
+LogTypes:
+    - Notion.AuditLogs
+RuleID: "Notion.Workspace.Public.Page.Added"
+Threshold: 1
+Tags:
```

Comment:
I love these tags and think they're relevant to this detection, but will ask for them to be removed for now as we're finalizing our threat detection use cases and will retroactively add these to their appropriate categories :)